eidas-saml-parser/src/main/java/uk/gov/ida/notification/eidassaml/EidasSamlResource.java

```diff
@@ -29,6 +30,12 @@
 @Produces(MediaType.APPLICATION_JSON)
 public class EidasSamlResource {
 
+    private static final Counter SUCCESSFUL_REQUESTS = Counter.build(
+            "verify_proxy_node_requests",
+            "Total number of successful Verify Proxy Node Requests")
+            .labelNames("issuer")
```

Comment:
do we know the list of issuers at app start-up? If so, it's good practice to "initialise" the counter with the list of possible `issuer` label values by calling `SUCCESSFUL_REQUESTS.labels(issuer);` for each `issuer` value.

The cost of not doing this is that you miss the first instance of each request in your counters on each app startup.